tests/test_events.js

```diff
@@ -15,11 +17,11 @@ test('emits request and replied events', function(t) {
     .reply(200)
 
   scope.on('request', function(req, interceptor) {
-    t.equal(req.path, '/please')
-    t.equal(interceptor.interceptionCounter, 0)
+    expect(req.path).to.equal('/please')
+    expect(interceptor.interceptionCounter).to.equal(0)
```

Comment:
I haven’t read the entire test (I’m on mobile and can’t see it) though wonder if this assertion is a good candidate for sinon-chai. Fine to defer that too, if you prefer.

Reply:
@paulmelnikow I've added `got` and `sinon` to these two tests. Want to take another look?